### ISSUE TITLE
Nest Thermostat connection blocked with fresh Adguard Home install

### ISSUE THREAD

**Issue by jack-fin** [AdguardTeam/AdGuardHome#3870]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  *  v0.106.3
* **How did you install AdGuard Home:**
  *  HomeAssistant Supervisor Add On
* **How did you setup DNS configuration:**
  *  I have it upstream to 1.1.1.1 and 8.8.8.8
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Expected Behavior

Fresh installation of Adguard Home with base configuration allows Nest Thermostat to connect to the internet

### Actual Behavior

Nest thermostat goes offline if I point my router to Adguard Home's DNS server, if I change it back to 1.1.1.1 directly Nest is able to connect.

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by flavorgold1** [comment]
Does AGH show any requests being made by the thermostat that are being blocked?

**Comment by ainar-g** [comment]
The thermostat could possibly be making requests to tracker companies.  Please inspect the query log page and filter it by client to see if that is the case.

**Comment by jack-fin** [comment]
This was due to a silly mistake on my end. I have a guest network at home. The Nest Thermostat has a terrible interface to enter the WiFi password so I connected it to my guest network with the shorter password.

The guest network does not have access to the Home Assistant IP address for DNS resolution, so the internet would fail.

I have added 1.1.1.1 as a fallback in my router if AdGuard Home is not available, this gets the guest network working and I suppose will make it so if AdGuard Home is not available, I will still have internet.
